chapters/Index.md

```diff
@@ -35,24 +35,2394 @@ Jintek           <info@jintek.consulting>
+
+1. How a ::BTCR2 Beacon:: can be established and added as a service to a DID document.  
+2. How ::BTCR2 Update Announcements:: are broadcast within ::Beacon Signals::.  
+3. How a resolver processes ::Beacon Signals::, identifying, verifying, and applying the authorized mutations to a DID document for a specific DID.
+
+This is an extensible mechanism, such that in the future new ::Beacon Types:: could be added.
+
+The current, active ::BTCR2 Beacons:: of a DID document are specified in the document's `service` property. By updating the DID document, a DID controller can change the set of ::BTCR2 Beacons:: they use to broadcast updates to their DID document over time. Resolution of a DID MUST process signals from all ::BTCR2 Beacons:: identified in the latest DID document and apply them in the order determined by the version specified by the `btcr2Update`.
+
+All resolvers of **did:btcr2** DIDs MUST support the ::Beacon Types:: defined in this specification.
+
+## Aggregation
+
+Three types of ::BTCR2 Beacons:: are defined: SingletonBeacon, MapBeacon and SMTBeacon.  Two of them, MapBeacon and SMTBeacon, support aggregation, i.e. the act of  committing to multiple ::BTCR2 Update Announcements:: in a ::Beacon Signal::.
+
+### Aggregation Participants
+
+The participants in aggregation are as follows:
+
+* DID controller \- A party that controls one or more **did:btcr2** identifiers participating in a ::BTCR2 Beacon::.  
+* ::Beacon Cohort:: \- The set of unique cryptographic keys participating in a ::BTCR2 Beacon:: that make up its n-of-n MuSig2 Bitcoin address.  
+* ::Beacon Aggregator:: \- The entity that coordinates the protocols of an Aggregate ::BTCR2 Beacon::, specifically the "Create Beacon Cohort" and "Announce Beacon Signal" protocols.  
+* ::Beacon Participant:: \- A member of a ::Beacon Cohort::, typically a DID controller, that controls cryptographic keys required to partially authorize the broadcasting of a ::Beacon Signal:: to the Bitcoin blockchain.  
+* Verifier \- A party verifying a **did:btcr2** identifier presentation.
+
+How coordination between an aggregator and multiple ::Beacon Participants:: is managed is out of scope, but one possible mechanism is outlined in “MuSig2 3-of-3 Multisig with Coordinator Facilitation” at [MuSig2 Sequence Diagrams](https://developer.blockchaincommons.com/musig/sequence/#musig2-3-of-3-multisig-with-coordinator-facilitation).
+
+When defining a ::Beacon Cohort::, the ::Beacon Aggregator:: may define the conditions for the cohort, including but not limited to:
+
+* Automatic publication to ::CAS:: (::Map Beacon:: only).  
+* Minimum and/or maximum number of ::Beacon Participants::.  
+* Minimum and/or maximum number of DIDs per ::Beacon Participant::.  
+* Cost of enrollment.  
+* Cost per signal per DID or ::Beacon Participant::.  
+* Minimum and/or maximum time between signals.  
+* Number of pending updates that trigger a signal.
+
+#### Singleton Beacon
```

Comment:
Certainly, this as a subsection of Aggregation participants feels wrong.

But even as a subsection of Aggregation doesn't seem right?